packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/cdk-hotswap-deployment-for-ecs-service-detects-failed-deployment-and-errors.integtest.ts

```diff
@@ -10,7 +10,7 @@ integTest(
 
     // WHEN
     const deployOutput = await fixture.cdkDeploy('ecs-hotswap', {
-      options: ['--hotswap'],
+      options: ['--hotswap', '--hotswap-ecs-stabilization-timeout-seconds', '10'],
```

Comment:
Decided not to do the `hotswap-ecs-stabilization-timeout=10s` option now because:

- The argument is also quite long even without the `-seconds` suffix.
- This can also be defined in `cdk.json` - so no excessive typing needed.